content/docs/iac/guides/building-extending/packages/packages.md

```diff
@@ -7,49 +7,77 @@ meta_image: /images/docs/meta-images/docs-meta.png
+- For packages without published SDKs, called [local packages](/docs/iac/guides/building-extending/packages/local-packages/#updating-local-packages), you can consume a package via the [`pulumi package add`](/docs/iac/cli/commands/pulumi_package_add/) command, which will download the package, either read (or dynamically generate) its schema, and generate a local SDK based on the schema file. The local SDKs may be committed to version control, or they can be regenerated at any time with the [`pulumi install`](/docs/iac/cli/commands/pulumi_install/) command.
 
-Regardless of the type of Pulumi Package you want to author, there are a few key steps in the process of authoring a Pulumi package.
+Some common use cases for local packages include:
 
-1. Decide the [type](#types-of-pulumi-packages) of package you want to create and create a repository for it using one of the template repos provided by Pulumi
-1. Create the [Resources or Components](/docs/concepts/resources/) you want to include in the package, either by authoring them manually (in the case of a Component Package) or generating them from a cloud provider's API or via a provider bridge
-1. Build the resource provider plugin: the binary file that contains all of the components or resources you defined in your source code
-1. Generate the SDK code for all languages supported by Pulumi and packs the SDK packages–the npm, NuGet, and Python packages–that the Pulumi Package’s users will reference in their own programs
-1. Publish the SDK packages and the resource provider plugin
+1. Using the Any Terraform provider to generate a local SDK for a Terraform provider. (This feature allows you to consume any Terraform provider in a Pulumi program.)
+1. Using the Azure Native provider to generate a local SDK for a specific version of the Azure API.
+1. Consuming a Pulumi component published in [Pulumi IDP](/docs/idp/).
 
-![A graphic representation of the steps listed above](../img/pulumi-package-concepts.png)
+In order to consume a Pulumi package, you must have the runtime installed for the language in which the provider is written. For example, if a package is written in TypeScript, any consumers of the package will require the NodeJS runtime to be installed in order to use the package.
 
-All Pulumi Packages must include a [schema](/docs/using-pulumi/pulumi-packages/schema/), which defines the resources and functions exposed by the package, and is used to drive the generation of language-specific SDKs and documentation.
+Go compiles to a native binary and does not require a runtime, and thus has the lowest overhead for package consumers. Most Pulumi packages in the Pulumi Registry (including all of the packages for the major cloud providers) are written in Go.
```

Comment:
This is only true for binary based plugins. Source based plugins will still need the Go compiler installed. Also dotnet plugins can be published as either runtime-dependent or runtime-included binaries (we'd recommend the latter). So what you need can get a bit fuzzy.

Reply:
Added the gory details. Would be awesome if you could check for accuracy. (Leaving this convo unresolved until I know the info I added is complete and accurate.)